v/vert.x/build-info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "vert.x",
+    "github_url":"https://github.com/eclipse-vertx/vert.x",
+    "version": "4.3.7",
+    "default_branch": "master",
+    "package_dir": "v/vert.x/", 
+    "build_script": "vertx_ubi_8.5.sh",
+    "docker_build": "false",
```

Comment:
@shreyakajbaje Could you please make all true or false values in build_info.json to boolean type.. e.g. "docker_build": false,